src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -368,9 +368,9 @@ def get_cont(self, pool, cont):
         query_response = self.daos_cmd.container_query(pool=pool.uuid, cont=cont)['response']
         cont_uuid = query_response['container_uuid']
 
-        # Convert default label to None
-        cont_label = query_response['container_label']
-        if cont_label == 'container_label_not_set':
+        if 'container_label' in query_response:
+            cont_label = query_response['container_label']
+        else:
             cont_label = None
```

Comment:
Minor/optional - Since `query_response` is a `dict`, and `dict.get()` returns `None` if not found:
```suggestion
        cont_label = query_response.get('container_label')
```

Reply:
@daltonbohning Just updated it as you suggested. Thank you very much!